### ISSUE TITLE
Identifying ' (single quote) fails

### ISSUE THREAD

**Issue by nc7s** [blueset/alfred-search-unicode#7]
… because

https://github.com/blueset/alfred-search-unicode/blob/15cf4f843095a4e92fb8a413e856f4dd2ea288d5/alfred-search-unicode/unicode_identify.py#L24

doesn't like `''` quoting itself:

```
[13:02:23.001] ERROR: Search Unicode[Script Filter] Code 1: Traceback (most recent call last):
  File "~/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.B54475EC-1187-4988-9D67-DF5AD3A6A816/unicode_identify.py", line 38, in <module>
    char, c_hex, c_int, utf8, xml, name, category = i
ValueError: not enough values to unpack (expected 7, got 6)
```

```
❯ ./uni -q identify "'" -f "%(char q),%(cpoint q),%(dec q),%(utf8 q),%(xml q),%(name q),%(cat q)"
''','U+0027','39','27','&#x27;','APOSTROPHE','Other_Punctuation'

❯ python -c 'import csv; print(list(csv.reader(["\'\'\',\'U+0027\',\'39\',\'27\',\'&#x27;\',\'APOSTROPHE\',\'Other_Punctuation\'"], quotechar="\'")))'
[["',U+0027'", '39', '27', '&#x27;', 'APOSTROPHE', 'Other_Punctuation']]
```

Now this is a curious case; do we special case `'''`? Ask upstream to properly check and escape `%(char q)`? Change to a more thoroughly designed upstream? Or, as I'm half interested in a PR, just bundle a Unicode data file and do the query ourselves, avoiding completely an unsigned binary?

**Comment by blueset** [comment]
Thanks for the report. Maybe having a special case for `'''` would be easier. Technically we can change upstream as the binary are compiled locally for Unicode 15, but I haven’t read _uni_’s source to see how complex it is.

**Comment by blueset** [comment]
Fixed in ddb5a92, released in v1.4.0.
